test/templates/button/index.njk

```diff
@@ -94,4 +94,19 @@
     {% endfor %}
 {% endfor %}
 
+
+{% set example %}
+
+<button class="fd-button fd-button--dropdown">Dropdown</button>
+<br>
+<button class="fd-button fd-button--compact fd-button--dropdown">Dropdown</button>
+<br>
+<button class="fd-button fd-button--dropdown full-width">Dropdown</button>
+<br>
+<button class="fd-button fd-button--dropdown full-width error">Dropdown</button>
```

Comment:
Don't "error" buttons just use the `fd-button--negative` class?

Reply:
![Screen Shot 2019-09-19 at 1 53 14 PM](https://user-images.githubusercontent.com/39598672/65268330-eb80db80-dae4-11e9-8de6-255d51ed2b8f.png)
![Screen Shot 2019-09-19 at 1 53 28 PM](https://user-images.githubusercontent.com/39598672/65268332-eb80db80-dae4-11e9-9710-b7072464102b.png)
Agreed, that was my original idea, but  `fd-button--negative` is a button with red text and red border, on hover it has red background and white color. In our case we only need a red border, so the amount of styling is less than overwriting the default behavior of `fd-button--negative`